### ISSUE TITLE
After removing package-lock.json the npm install breaks

### ISSUE THREAD

**Issue by peterschriever** [hyva-themes/magento2-hyva-checkout#101]
If trying to regenerate a clean package-lock.json the `npm install` command breaks because of an inconsistent eslint dependency;

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: hyva-checkout@0.1.0
npm ERR! Found: eslint@6.8.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^6.8.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">=7.0.0" from eslint-config-prettier@7.2.0
npm ERR! node_modules/eslint-config-prettier
npm ERR!   dev eslint-config-prettier@"^7.1.0" from the root project
```

**Comment by progammer-rkt** [comment]
I tried this now by removing `node_modules` and `package-lock.json` and I didn't face any issues.

**Comment by progammer-rkt** [comment]
When I tried to perform `npm update`, I encountered this issue. The fix is available in #130

**Comment by progammer-rkt** [comment]
The fix reached `develop` branch and hence closing this issue
